scipy/stats/stats.py

```diff
@@ -7189,9 +7199,168 @@ def mannwhitneyu(x, y, use_continuity=True, alternative=None):
     return MannwhitneyuResult(u, p)
 
 
+# TODO: add paired=False, do a signed-rank test if paired=True or y is
+#       not provided. See morestats.wilcoxon
+def mann_whitney_u(x, y, correction=True, exact='auto', alternative='two-sided'):
```

Comment:
Detailed review of this function not needed. Many remaining comments have been addressed in `mannwhitneyu2` above; I'm just leaving this for comparison and to decide whether there are any parts of it we want to keep.